SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -343,11 +343,36 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Runtime", AdvancedDisplay)
 	FRuntimeVariantVersion CompatibilityModeRuntimeVersion;
 
+	/** Whether to use the GDK-associated SpatialOS inspector version for local deployments, or to use the one specified in the
+	 * InspectorVersion field. */
+	UPROPERTY(EditAnywhere, config, Category = "Inspector", meta = (DisplayName = "Use GDK Pinned Inspector Version"))
+	bool bUseGDKPinnedInspectorVersion = true;
+
+	/** Runtime version to use for local deployments, if not using the GDK pinned version. */
+	UPROPERTY(EditAnywhere, config, Category = "Inspector", meta = (EditCondition = "!bUseGDKPinnedInspectorVersion"))
+	FString InspectorVersionOverride;
+
+	FString InspectorPinnedVersion;
```

Comment:
This doesn't need to exist here. Can just use `SpatialGDKServicesConstants::SpatialOSInspectorPinnedVersion` below.